linkerd/protocol/http/src/integration/scala/io/buoyant/linkerd/protocol/TlsBoundPathTest.scala

```diff
@@ -212,8 +212,8 @@ class TlsBoundPathTest extends FunSuite with Awaits {
   }
 
   test("wrong common name") {
-
-    withCerts("bill.buoyant.io", "ted.buoyant.io") { certs =>
+    // Create certificates without SAN DNS entries that override CN.
+    withCertsWithCustomDnsAltNames(Seq("bill.buoyant.io", "ted.buoyant.io"), null) { certs =>
```

Comment:
Not so happy with this method name, if you have a better name suggestion, plz recommend.